packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -291,6 +295,13 @@ export interface ISemanticColors {
    */
   linkHovered: string;
 
+  //// Keytips
+
+  /**
+   * The background color for a disabled keytip.
+   */
+  keytipBackgroundDisabled: string;
```

Comment:
it is weird that there is a background for a disabled keytip but nothing else...